core/net/ipv6/uip-nd6.c

```diff
@@ -662,7 +662,8 @@ rs_input(void)
         }
         if(memcmp(&nd6_opt_llao[UIP_ND6_OPT_DATA_OFFSET],
             lladdr, UIP_LLADDR_LEN) != 0) {
-          uip_ds6_nbr_t nbr_data = *nbr;
+          uip_ds6_nbr_t nbr_data;
+          nbr_data = *nbr;
```

Comment:
On my machine, running [latest git  cc65](https://github.com/cc65/cc65):

```c
typedef struct {
	int a, b, c;
} test_type;

void test() {
	test_type *pointer;
	test_type value;
	value = *pointer;
}
```
Compiles fine, while:
```c
typedef struct {
	int a, b, c;
} test_type;

void test() {
	test_type *pointer;
	test_type value = *pointer;
}
```
Gives:
```
test.c(7): Error: `{' expected
test.c(7): Error: Too many local variables
test.c(7): Error: Constant expression expected
test.c(7): Error: `}' expected
```
Am I using an unsupported cc65? Which one should be used with Contiki?